site/fs.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs').promises
+const cpy = require('cpy')
+const rimraf = require('rimraf')
+const util = require('util')
+
+const rimrafAsync = util.promisify(rimraf)
+
+const copyDirRecursiveAsync = (src, dest) => cpy(src, dest)
+
+const ensurePathExistsAsync = async path => {
+  try {
+    await fs.mkdir(path, { recursive: true })
+  } catch (_) {
```

Comment:
`fs.mkdir()` [does not throw](https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback) when `path` does not exist providing `recursive: true` is passed.

Reply:
good to know!
I will remove the try/catch and rebase with master.